hudi-common/src/main/java/org/apache/hudi/common/model/HoodieColumnRangeMetadata.java

```diff
@@ -30,16 +28,21 @@
   private final String columnName;
   private final T minValue;
   private final T maxValue;
-  private final long numNulls;
-  private final PrimitiveStringifier stringifier;
+  private final long nullCount;
+  private final long valueCount;
+  private final long totalSize;
+  private final long totalUncompressedSize;
 
-  public HoodieColumnRangeMetadata(final String filePath, final String columnName, final T minValue, final T maxValue, final long numNulls, final PrimitiveStringifier stringifier) {
+  public HoodieColumnRangeMetadata(final String filePath, final String columnName, final T minValue, final T maxValue,
```

Comment:
should we rename this class to HoodieColumnStatsMetadata or something.

Reply:
HoodieColumnRangeMetadata  is an existing class. ParquetUtils and ColumnStatsIndexHelper use them widely. Don't prefer to rename this class in the scope of this PR.